runtests.py

```diff
@@ -0,0 +1,15 @@
+
+import os
+import sys
+
+import django
+from django.conf import settings
+from django.test.utils import get_runner
+
+if __name__ == "__main__":
+    os.environ['DJANGO_SETTINGS_MODULE'] = 'tests.settings'
+    django.setup()
+    TestRunner = get_runner(settings)
+    test_runner = TestRunner(verbosity=1, interactive=True)
+    failures = test_runner.run_tests(["geospaas_rest_api"])
+    sys.exit(bool(failures))
```

Comment:
Add line break.
Try to set it up automatically in VS.

Reply:
could you please explain more about this comment? I don't understand it exactly. I think it is simple but I might need more explanations.